guide/src/class.md

```diff
@@ -272,6 +272,8 @@ However, because of some technical problems, we don't currently provide safe upc
 that inherit native types. Even in such cases, you can unsafely get a base class by raw pointer conversion.
 
 ```rust
+# #[cfg(Py_LIMITED_API)] fn main() {}
```

Comment:
Subclassing a builtin type (`PyDict`)

Reply:
Ah yes, of course. It could be worth adding a comment in this example which says this only works with the unlimited API. (Also the paragraph above starting on line 268 could also benefit from a similar statement saying inheriting native types is not possible with the limited api.)